lib/travis/api/v3/queries/build.rb

```diff
@@ -33,5 +35,12 @@ def restart(user)
         payload
       end
     end
+
+    def priority(user)
+      raise NotFound, "Job not found" if find.jobs.blank?
+      build_job = find.jobs.find_by_commit_id(find.commit_id)
```

Comment:
@dmaity-celestialsys, the build may contain multiple jobs (one-to-many relation, see [Build#matrix](https://github.com/travis-ci/travis-api/blob/master/lib/travis/api/v3/models/build.rb#L17-L21) for more details). We need to update `priority` in all jobs that are related to this build.

Reply:
Hi @pavel-d, I have updated code to set priority for multiple jobs those are associated with particular build.